ekncontent/ekncontent/eknc-domain.c

```diff
@@ -820,6 +908,16 @@ on_xapian_query_response (GObject *source,
       return;
     }
 
+  state->models = g_ptr_array_new_full (state->total_models, maybe_unref_object);
+  g_ptr_array_set_size (state->models, state->total_models);
```

Comment:
There should be no reason to call this here -- even more so because the previous line already sets the size of the array to `state->total_models`